tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -315,3 +323,17 @@ datasets:
     preprocessing:
       - type: resize
         size: 224
+
+  - name: affectnet
```

Comment:
please, no modify this file, it is for light-accuracy validation on 10 images only (OSS OMZ CI). in internal validation, we use internal definitions file as previously with extended set of datasets

Reply:
@eaidova done